apps/public-docsite-v9/src/Concepts/Migration/FromV8/ComponentMapping.stories.mdx

```diff
@@ -55,37 +55,37 @@ Check out the latest schedule information on the [v9 Component Roadmap](https://
 | ProgressIndicator      | [ProgressBar](/docs/components-progressbar--default)                                                                                         |
-| Rating                 | _[In Progress](https://github.com/orgs/microsoft/projects/786/views/1?pane=issue&itemId=24403184)_                                           |
+| Rating                 | [Rating](/docs/components-rating--default)                                                                                                   |
 | ResizeGroup            |                                                                                                                                              |
 | ScrollablePane         |                                                                                                                                              |
-| SearchBox              |                                                                                                                                              |
+| SearchBox              | [SearchBox](/docs/components-searchbox--default)                                                                                             |
 | Separator              | [Divider](/docs/components-divider--default)                                                                                                 |
 | Shimmer                | [Skeleton](/docs/components-skeleton--default)                                                                                               |
 | Slider                 | [Slider](/docs/components-slider--default)                                                                                                   |
 | SplitButton            | [Menu with SplitButton as the Menu Trigger](/docs/components-button-splitbutton--default)                                                    |
-| SpinButton             | [~SpinButton](/docs/components-spinbutton--default)                                                                                          |
+| SpinButton             | [SpinButton](/docs/components-spinbutton--default)                                                                                           |
 | Spinner                | [Spinner](/docs/components-spinner--default)                                                                                                 |
 | Stack                  | [Migration Guide](/docs/concepts-migration-from-v8-components-flex-stack--page), [StackShim](/docs/migration-shims-v8-stackshim--playground) |
 | SwatchColorPicker      | [SwatchPicker](/docs/components-swatchpicker--default)                                                                                       |
 | TagPicker              |                                                                                                                                              |
```

Comment:
@tudorpopams I think we should link this to [TagPicker](https://react.fluentui.dev/?path=/docs/components-tagpicker--default)

Reply:
Updated.